src/DataProtection/AzureStorage/test/Microsoft.AspNetCore.DataProtection.AzureStorage.Tests.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp2.2;net461</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <UseLatestPackageReferences>true</UseLatestPackageReferences>
```

Comment:
@mikaelm12 any issue with this coming into 'release/2.2' from 'release/2.1' i.e. was it just unnecessary?